data-processing/pulsar-flink/2.7.0/pulsar-flink.md

```diff
@@ -200,6 +200,49 @@ In some scenarios, users need messages to be strictly guaranteed message order t
 
 Pulsar Flink connector supports this feature as well. This feature can be enabled by configuring the `enable-key-hash-range=true` parameter. When enabled, the range of Key Hash processed by each consumer is divided based on the parallelism of the task.
 
+## Fault tolerance
+
+Pulsar Flink connector 2.7.0 provides different semantics for source and sink.
```

Comment:
I think we can remove  2.7.0 since the doc is specific for 2.7.0.

Reply:
There are some differences between 2.4.1 and 2.7.0, so I've provided specific versions.

![image](https://user-images.githubusercontent.com/50226895/104435448-5e11a680-55c7-11eb-9695-2f217ee5a5e7.png)